README_BOV.md

```diff
@@ -0,0 +1,91 @@
+# bov
```

Comment:
We can rename to `bcpd` here

Reply:
Great. I was wondering about naming...
This will break some of the Docker scripts as well, so maybe they can fix that up once we have a new name.